x-pack/qa/smoke-test-watcher-with-security/custom-log4j2.properties

```diff
@@ -0,0 +1,278 @@
+#######################################
+# custom properties
+#######################################
+
+logger.watcher.name = org.elasticsearch.xpack.watcher
```

Comment:
I think it's enough to say `setting 'logger.org.elasticsearch.xpack.watcher', 'debug'` etc. instead of adding an entire extra configuration file?

Reply:
thanks @jasontedor , i wasn't aware that was possible.  tested it and works great. fixed on https://github.com/elastic/elasticsearch/pull/42092/commits/119322cad12ca080dac9995b663dbd35e9903c4d